apps/admin-portal/src/pages/local-claims.tsx

```diff
@@ -235,30 +236,28 @@ export const LocalClaimsPage = (): ReactElement => {
                                         {
                                             key: 0,
                                             text: t("common:name"),
-                                            value: "name"
+                                            value: "displayName"
                                         },
                                         {
                                             key: 1,
-                                            text: t("common:description"),
-                                            value: "description"
+                                            text: "Attribute URI",
+                                            value: "claimURI"
                                         }
                                     ] }
                                     filterAttributePlaceholder={
-                                        t("devPortal:components.userstores.advancedSearch.form.inputs" +
-                                            ".filterAttribute.placeholder")
+                                        "E.g. Name, Attribute URI etc."
```

Comment:
Translations are available for these strings.
Check under `devPortal:components.claims.local.advancedSearch`